libs/core/src/lib/command/clean-stack.ts

```diff
@@ -1,7 +1,7 @@
 import { ExecaError } from "execa";
 
-export function cleanStack(err: ExecaError, className: string) {
-  const lines = err.stack ? err.stack.split("\n") : String(err).split("\n");
+export function cleanStack(err: string | ExecaError, className: string) {
+  const lines = isAExecaError(err) && err.stack ? err.stack.split("\n") : String(err).split("\n");
```

Comment:
Shouldn't isExecError(err) replace `err.stack` here, rather than be checked in addition to it, based on how you have implemented it?

Reply:
Changed the interface to error as there is nothing specific needed from ExecaError. I renamed typeguard to make it clear what is checked.